rasa/core/broker.py

```diff
@@ -220,3 +220,74 @@ def _publish(self, event):
 
     def _close(self):
         self.producer.close()
+
+
+class SQLProducer(EventChannel):
```

Comment:
@tmbo should that be part of this PR 😄 ?

Reply:
thanks for catching and no 😅